hw4/hw4.py

```diff
@@ -1,40 +1,27 @@
+# Continuous or Boolean
+# Loop through each predictor
+# Determine if the predictor is cat/cont
+# Automatically generate the necessary plot(s) to inspect it
+# Calculate the different ranking algos
+# p-value & t-score (continuous predictors only) along with it's plot
+
+
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import statsmodelsgit .api as sm
-from plotly import figure_factory as ff
-from plotly import graph_objects as go
-from scipy import stats
-from sklearn import datasets, linear_model
-from sklearn.linear_model import LinearRegression
-from sklearn.metrics import confusion_matrix
-
-# Diabetes
-
-
-# diabetes = datasets.load_diabetes(as_frame=True)
-# df = diabetes['frame']
-# predictors = ['age','sex','bmi','bp','s1','s2','s3','s4','s5','s6']
-# response = ['target']
-
-# Iris
-df = pd.read_csv("iris.csv")
-df.head()
-predictors = ["Id", "SepalLengthCm", "SepalWidthCm", "PetalLengthCm", "PetalWidthCm"]
```

Comment:
Good Idea getting rid of the hard coded predictors. I think the idea for this assignment is that we want to create a dynamic program that can handle any pandas data frame. Which suggests  the less hard coded variables the better